rigex-front-end/src/constants/typeDefs.js

```diff
@@ -6,6 +6,7 @@ export const typeDefs = gql`
     getAllAboutText: [AboutText]
     getAllServices: [Services]
     getAllRigs: [Rigs]
+    getAallTeamMembers:  [TeamMembers]!
```

Comment:
Typo maybe? getAallTeamMembers don't seem right...

Reply:
My keyboard has been dying and doing weird things, typo is the correct answer